lib/sdk.js

```diff
@@ -0,0 +1,414 @@
+var blocks = require('block-stream2');
+var bz2 = require('unbzip2-stream');
+var createHash = require('sha.js');
+var fs = require('fs-extra-promise');
```

Comment:
t2-cli is using `fs-extra` exclusively everywhere else—why does this file need to have a different thing?